src/main/java/com/coffeine/virtuoso/music/view/form/TextForm.java

```diff
@@ -22,16 +22,14 @@
 public class TextForm {
 
     /// *** Properties  *** ///
-    protected Long id;
-
     @NotNull
     @Min( 0 )
-    protected Long songId;
+    private Long songId;
 
     @NotNull
     @NotEmpty
     @Length( max = 5 )
-    protected String locale;
+    private String locale;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/vitaliyacm/Virtuoso/file/1555529868/issues/source?bid=2991943&fileBranchId=3017828#l32)